packages/components/src/tests/commonTests/disabled.ts

```
@@ -190,7 +190,9 @@ export function disabled(componentTestSetup: ComponentTestSetup, options?: Disab
     await waitForAnimationFrame(page);
     await waitForAnimationFrame(page);
 
-    await expectToBeFocused(page, effectiveFocusTarget.click.pointer, "click");
+    if (tag !== "calcite-flow-item") {
```

FIXME: Let's try to address the test failure instead of skipping.

---

Agreed, I tried to convey this in the commit message, just wanted to test if the rest of the test would pass if this line didn't execute, and it does pass, so that's good to know.  I'm restoring it in the next commit.